### DISCUSSION TITLE
Go to the source: list of the repository of each `plantuml` on each GNU/Linux distro.

### DISCUSSION THREAD

**Discussion by The-Lum** [plantuml/plantuml#1572] [Q&A]
Hi all,

- Where are the repositories or links to all the `plantuml` version on each GNU/Linux distributions?
_Especially main distro. like `Debian`, `Ubuntu`, `Fedora`, ...?_

_[Main objective: in order to know the different works on plantuml distro., make list, profit the works and fix/patch, ,...]_

Regards,
Th.

**Comment by The-Lum** [comment]
See first attempt with:
- #1693

**Comment by rzr** [comment]
I have been tracking this page:

https://repology.org/project/plantuml/

And a bit annoyed that debian is outdated, do you suggest to pull using a different package manager? yes which one, I am considering nix but there should be something more javaish is there ?
